src/utils.coffee

```diff
@@ -44,13 +44,18 @@ class Utils
     result = [result] unless result instanceof Array
     _.compact(result)
 
-  @wrapObjects: (array) ->
+  @wrapObjects: (array, options = {}) ->
     output = []
     _(array).each (elem) =>
       if elem.constructor == Object
         for key, value of elem
           o = {}
-          o[key] = @wrapObjects(if value instanceof Array then value else [value])
+
+          if options.ignoreCollections && elem[key] instanceof Backbone.Collection
```

Comment:
The problem here is `include: { 'foo': [someCollection] }` will get converted to: 
![screen shot 2017-05-08 at 11 55 32 am](https://cloud.githubusercontent.com/assets/2433091/25820186/4920ef5e-33e5-11e7-8c0d-b949b53c1ae7.png)